app/src/main/java/io/github/sds100/keymapper/mappings/keymaps/trigger/KeyMapTrigger.kt

```diff
@@ -165,3 +166,54 @@ object KeymapTriggerEntityMapper {
         )
     }
 }
+
+// Comparator.reversed() requires API level 24
+class KeyMapTriggerComparator(
```

Comment:
Should this not be called KeyMapComparator? Can you put it inside the KeyMap file?

Reply:
Yeah name and location is a bit off. There are comparators  for every `SortField` and they are used for sorting `KeyMap` by given field.

`KeyMapTriggerComparator`, `KeyMapActionsComparator`, `KeyMapConstraintsComparator`, `KeyMapOptionsComparator` 
All of them are located kinda randomly.

Reply:
If there isn't a good correspondence with the data classes then move the comparators to individual files in side a `sorting/comparators` package?